client/history/client.go

```diff
@@ -460,3 +476,16 @@ redirectLoop:
 	}
 	return err
 }
+
+func aggregateYarpcOptions(ctx context.Context, opts ...yarpc.CallOption) []yarpc.CallOption {
```

Comment:
This method is defined twice, can you move it to common?

Reply:
done